packages/tiler-sharp/src/index.ts

```diff
@@ -80,81 +84,28 @@ export class TileMakerSharp implements TileMaker {
     return this.toImage(format, this.createImage(background).composite(layers));
   }
 
-  /** Can we just serve the image directly from the cotar */
-  private isDirectCotar(ctx: TileMakerContext): boolean {
```

Comment:
these paths are being hit 0.01% of the time and barely save any time, remove for simplication